tests/include/ApiTest.php

```diff
@@ -1273,8 +1273,10 @@ public function testApiMediaUploadWithValidMedia()
 	 * Test the api_status_show() function.
 	 * @return void
 	 */
-	public function testApiStatusShow()
+	public function testApiStatusShowWithJson()
 	{
+		$this->markTestSkipped('This test requires an item ID');
```

Comment:
There are 6 items at
https://github.com/friendica/friendica/blob/develop/tests/datasets/api.yml

Cannot we take one of them?